### ISSUE TITLE
could not extract over 2GB PBP

### ISSUE THREAD

**Issue by xenyou** [has207/psxtract-2#10]
it seems that psxtract-2 can't handle huge PBP file (over 2GB)

I got this error.

```
PS D:\psxtract-2> .\psxtract.exe -c ..\backups20240502diff\game\NPJJ00307\USRDIR\CONTENT\EBOOT.PBP
Unpacking PBP ..\game\NPJJ00307\USRDIR\CONTENT\EBOOT.PBP...

UNPACK_PBP ERROR: Could not get the input file size.
```

The game is `tokimeki memorial 2`.  Its size is 2241040KB.

This game also includes `__sce_discinfo` file in CONTENT directory (attached file).
I don't know but It might be help for deviding data. 

[__sce_discinfo.zip](https://github.com/has207/psxtract-2/files/15183404/__sce_discinfo.zip)

**Comment by has207** [comment]
thanks for the report, I'll check it out

**Comment by Tokman5** [comment]
I had encountered the same issue about the same game(`Tokimeki Memorial 2`).
Because its size is beyond 32bit signed range, I needed to make some changes for variable type and functions (check my commit https://github.com/Tokman5/psxtract/commit/8756a60bfbbc19d915ff2b5b72fa3354b39e36d3 ).
After doing that, converting the game did successfully and md5 hashes of all 5 discs matched the ones in redump.org.
